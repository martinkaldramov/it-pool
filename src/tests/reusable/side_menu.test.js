import React from 'react';
import { shallow } from 'enzyme';
import SideMenu from '../../components/reusable/side_menu';

describe('SideMenu', () => {
  const wrap = shallow(<SideMenu />);

  it('should render successfully', () => {
    expect(wrap).toMatchSnapshot();
  });
});
