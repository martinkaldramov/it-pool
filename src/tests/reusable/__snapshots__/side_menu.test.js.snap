// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideMenu should render successfully 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SideMenu />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <MenuItem
          anchorOrigin={
                    Object {
                              "horizontal": "right",
                              "vertical": "top",
                            }
          }
          checked={false}
          desktop={false}
          disabled={false}
          focusState="none"
          insetChildren={false}
          targetOrigin={
                    Object {
                              "horizontal": "left",
                              "vertical": "top",
                            }
          }
>
          <ActionHome />
          Home
</MenuItem>,
        <Divider
          inset={false}
/>,
        <StaticMenuItems />,
        <Divider
          inset={false}
/>,
        <RecentSearches />,
        <Divider
          inset={false}
/>,
      ],
      "disableSwipeToOpen": false,
      "docked": true,
      "open": null,
      "openSecondary": false,
      "swipeAreaWidth": 30,
      "width": null,
      "zDepth": 2,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "anchorOrigin": Object {
            "horizontal": "right",
            "vertical": "top",
          },
          "checked": false,
          "children": Array [
            <ActionHome />,
            "Home",
          ],
          "desktop": false,
          "disabled": false,
          "focusState": "none",
          "insetChildren": false,
          "targetOrigin": Object {
            "horizontal": "left",
            "vertical": "top",
          },
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "Home",
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "inset": false,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "inset": false,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "inset": false,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <MenuItem
            anchorOrigin={
                        Object {
                                    "horizontal": "right",
                                    "vertical": "top",
                                  }
            }
            checked={false}
            desktop={false}
            disabled={false}
            focusState="none"
            insetChildren={false}
            targetOrigin={
                        Object {
                                    "horizontal": "left",
                                    "vertical": "top",
                                  }
            }
>
            <ActionHome />
            Home
</MenuItem>,
          <Divider
            inset={false}
/>,
          <StaticMenuItems />,
          <Divider
            inset={false}
/>,
          <RecentSearches />,
          <Divider
            inset={false}
/>,
        ],
        "disableSwipeToOpen": false,
        "docked": true,
        "open": null,
        "openSecondary": false,
        "swipeAreaWidth": 30,
        "width": null,
        "zDepth": 2,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "anchorOrigin": Object {
              "horizontal": "right",
              "vertical": "top",
            },
            "checked": false,
            "children": Array [
              <ActionHome />,
              "Home",
            ],
            "desktop": false,
            "disabled": false,
            "focusState": "none",
            "insetChildren": false,
            "targetOrigin": Object {
              "horizontal": "left",
              "vertical": "top",
            },
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "Home",
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "inset": false,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "inset": false,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "inset": false,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
